tests/UKHO.ERPFacade.API.FunctionalTests/Configuration/TestConfig.cs

```diff
@@ -15,6 +15,12 @@ public class TestConfig
         public string GeneratedProductJsonFolder { get; set; }
         public string GeneratedJSONFolder { get; set; }
         public SharedKeyConfiguration SharedKeyConfiguration { get; set; }
+        public string LicenceUpdate { get; set; }
+        public string[] ROSLUXMLList { get; set; }
```

Comment:
Can we have meaningful name here? E.g RosLicenceUpdateXMLList

Reply:
Done